### PR TITLE
Set content picks default title/mime if file is created for the first time

### LINKED ISSUE #272: #223 silently renames file on update by ID
I don't know if this issue counts as a bug, but at least in my case it breaks file updates and I had to search a lot to figure out what was going on.

With v1.15.1, when I update a file using only its ID (like below), the file is silently renamed from `file.txt` to `/path/to/file.txt`.

```python
conf = {
    'id': file_id,
    'mimeType': 'application/zip',
    'parents':[{
        'kind': 'drive#fileLink', 
        'id': parent_id
    }]
}

gf = self.gdrive.CreateFile(conf)
gf.SetContentFile('/path/to/file.txt') 
gf.Upload()
```

This does not happen when using v1.14.0.
I think it is a side effect of #223 and caused by the following in `def SetContentFile(self, filename):`

```python
if self.get("title") is None:
    self["title"] = filename
```

A simple solution is of course to set the file name explicitly.

### PR DESCRIPTION
**Pull request by shcheklein**
Fixes #272 . Followup after #273 . 

Trying a bit less dramatic change. It should not be breaking backward compatibility and is fixing #272 as a regression.

### CODE REVIEW

**Comment by shcheklein** [comment]
hey @efiop sorry for bothering you, decided that it can be done in a bit less aggressive mode and probably would just a regression fix vs a incompatible change.

File: docs/filemanagement.rst
Comment (efiop) [inline docs/filemanagement.rst:1]: Not sure I understand what was modified here. Did linter modify something unrelated?

Comment (shcheklein) [reply docs/filemanagement.rst:1]: yep, unrelated (was looking if something needs to be changed and editor modified it - I think it's fine to keep it as a cleanup)

Comment (efiop) [reply docs/filemanagement.rst:1]: @shcheklein This needs to be either modified consistently everywhere (with pre-commit) or not touched at all. Your editor is just your editor, please do not include unrelated changes like these.

**Comment by efiop** [comment]
Side note: please consider including the "subsystem" name that you are modifying in the PR title. https://github.com/iterative/PyDrive2/pull/273 should've been something like `files: don't magically set filename on set file content`. The current PR title is completely not descriptive. Also `fix(123)` stuff is kinda useless in PR titme as it is not informative (issue mention in the PR description is informative).

File: docs/filemanagement.rst
Comment (shcheklein) [reply docs/filemanagement.rst:1]: yep, agreed. I still it's fine to do it here. Not critical for this repo. We can setup linter as a followup.

**Comment by shcheklein** [comment]
>I'm not sure what conventions you are following in pydrive2

@efiop it loosely based on this https://www.conventionalcommits.org/en/v1.0.0-beta.2/ . But it's not required here. It's not set as a standard or a requirement. 

I agree that it can be better - I'll update it

File: docs/filemanagement.rst
Comment (shcheklein) [reply docs/filemanagement.rst:1]: just to clarify, it's not only linter in this case (there was a minor bug fix `can can` -> `can`) in the relevant docs. It doesn't make it relevant, but make it a justifiable cleanup of the relevant docs file in this case. Let me know if it's a strong opinion on your end - I can certainly move it to a separate PR.

Comment (efiop) [reply docs/filemanagement.rst:1]: We are spending more time discussing this than it would take to just exclude that file from this PR 😄

Comment (shcheklein) [reply docs/filemanagement.rst:1]: Hmm, okay. Though I don't think it's a good practice to do changes on someone else's PR w/o an agreement first. You can do a PR (to a PR) in this case and an owner can accept or not I think. That would be better imo. Anyways, thanks for the review.

Comment (efiop) [reply docs/filemanagement.rst:1]: No functional changes, only removing unrelated changes that we both agree are unrelated.

**Comment by efiop** [comment]
> It's not set as a standard or a requirement.

Not setting it as a requirement for everyone. Just thought we at least were in sync between us, but looks like we are not 🙁 Noted and I won't pay as much attention to it next time 👍

**Comment by shcheklein** [comment]
@efiop nw. It was based on the _commit_ standard since I didn't pay enough attention and PR title got auto-generated from a commit message (that was bad by itself since I planned to amend it later- I was literally first trying things around and forgot to amend it). So, to clarify I don't think it was good, or that I like that standard for PR titles - it probably fits in some way, and if we use squash it becomes indeed somewhat useful to use that standard, but I'm totally fine with a regular DVC standard as well.

Standards aside - title was not descriptive anyways, and was not following the (any) standard anyways, so good catch. I think this is the most important part here. Details don't matter much (to me).

**Pull request merged**